src/screens/LoadingScreen/index.js

```diff
@@ -26,11 +31,20 @@ const LoadingScreen = () => {
         checkIfLoggedIn()
       }
     }
-    decideIfGoToPrivacyPolicyScreenOrLoginScreen()
+
+    setTimeout(() => decideIfGoToPrivacyPolicyScreenOrLoginScreen(), 4000)
```

Comment:
My question is, do we really need those 4 seconds? I think the faster the app loads, the better. Even if the screen is not shown at all because no loading happens, it's better.

@MPucheta